common/configure

```diff
@@ -196,7 +196,6 @@ addInstallFile "../AUTHORS"          "/share/doc/backintime-common"
 addInstallFile "../LICENSE"          "/share/doc/backintime-common"
 addInstallFile "../README.md"        "/share/doc/backintime-common"
 addInstallFile "../TRANSLATIONS"     "/share/doc/backintime-common"
-addInstallFile "../VERSION"          "/share/doc/backintime-common"
```

Comment:
Should we keep this file in the installation target for "backwards compatibility" (though I am not sure if anyone is using it at all)

Reply:
I am 99% sure. ;) BIT does not use it.
I could ask at Debian and Arch if the distro maintainers do use it. But I assume they are not.
My assumption is that this file appeared in the release just out of habbit.